awx/main/scheduler/task_manager.py

```diff
@@ -471,9 +475,8 @@ def after_lock_init(self):
         Init AFTER we know this instance of the task manager will run because the lock is acquired.
         """
         self.dependency_graph = DependencyGraph()
-        self.instances = TaskManagerInstances(self.all_tasks)
-        self.instance_groups = TaskManagerInstanceGroups(instances_by_hostname=self.instances)
-        self.controlplane_ig = self.instance_groups.controlplane_ig
+        self.tm_models = TaskManagerModels()
+        self.controlplane_ig = self.tm_models.instance_groups.controlplane_ig
```

Comment:
super duper minor comment, but I'd prefer to save `controlplane_ig` as a local variable in process_pending_tasks instead of `self.controlplane_ig`.

But the semantic of `self.tm_models` is an improvement from before, this is going in the right directions.